.github/workflows/cypress.yml

```diff
@@ -0,0 +1,156 @@
+name: Cypress Tests
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - alpha
+      - beta
+      - stable
+
+env:
+  CYPRESS_TEST_USER_EMAIL: "dev@mr-beam.org"
+  CYPRESS_TEST_USER_PASSWORD: "a"
+  MRBEAM_PLUGIN_URL: "http://plugin-address.org:5002/"
+  MRBEAM_PLUGIN_DOMAIN_NAME: "plugin-address.org"
+
+jobs:
+  mrb-docker:
+    runs-on: ubuntu-latest
+    outputs:
+      domain: ${{ env.MRBEAM_PLUGIN_DOMAIN_NAME }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build and export
+        uses: docker/build-push-action@v3
+        with:
+          push: false
+          context: .
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          file: ./Dockerfile
+          tags: mrbeam_plugin:latest
+          outputs: type=docker,dest=/tmp/image.tar
+      - name: Upload MrBeamPlugin image
+        uses: actions/upload-artifact@v3
+        with:
+          name: mrbeam_plugin
+          path: /tmp/image.tar
+
+  ui-chrome-tests:
+    runs-on: ubuntu-latest
+    permissions: write-all
+    container:
+      image: cypress/browsers:node16.17.0-chrome106
+      options: --add-host=${{ needs.mrb-docker.outputs.domain }}:host-gateway
+      volumes:
+        - /var/run/docker.sock:/var/run/docker.sock
+    needs: mrb-docker
+    strategy:
+      fail-fast: false
+      matrix:
+        # run copies of the current job in parallel
+        containers: [ 0, 1, 2, 3, 4 ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Download MrBeamPlugin image
+        uses: actions/download-artifact@v3
+        with:
+          name: mrbeam_plugin
+          path: /tmp
+      - name: Load Docker image
+        run: |
+          apt-get --allow-releaseinfo-change update
+          apt-get install docker.io -y
+          docker load --input /tmp/image.tar
+          docker run --name mrbeam-plugin -d -p5002:5000 mrbeam_plugin
+      - run: |
+          envsubst < ./cypress/fixtures/test-data.template.json > ./cypress/fixtures/test-data.json
+          envsubst < ./cypress.config-template.js > ./cypress.config.js
```

Comment:
should we also include a default cypres.config.js for local usage? I guess that's already part of the repository? 🤔  same for test-data

Reply:
Yes, we have both :smiley: